darc/darc.go

```diff
@@ -1279,10 +1278,15 @@ func evalThreshold(visited map[string]bool, getDarc GetDarc,
 		}
 	}
 
-	computedFraction := float64(len(validIds)) / float64(len(uniqIds))
-	if computedFraction < threshold {
-		return xerrors.Errorf("computed fraction is lower than threshold: %f < %f",
-			computedFraction, threshold)
+	// we floor the number of identities to 1, so we avoid 0/0.
```

Comment:
nitpicking :smile: 
I am not sure "floor" is the proper term here, as it usually means "rounding to the largest integer smaller than"... maybe "we force ... to be at least 1"? Like I said, nitpicking :laughing:

Reply:
(single-sided) clamping then?

Reply:
Kotlin has the `coerceAtLeast()`, `coerceAtMost()` and `coerceIn()` [functions](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/#functions) for this purpose...